src/coreclr/vm/eventing/eventpipe/ds-rt-coreclr.h

```diff
@@ -320,6 +321,35 @@ ds_rt_profiler_startup (DiagnosticsStartupProfilerCommandPayload *payload)
 }
 #endif // PROFILING_SUPPORTED
 
+static
+uint32_t
+ds_rt_set_environment_variable (const ep_char16_t *name, const ep_char16_t *value)
+{
+	return SetEnvironmentVariableW(reinterpret_cast<LPCWSTR>(name), reinterpret_cast<LPCWSTR>(value)) ? S_OK : HRESULT_FROM_WIN32(GetLastError());
+}
+
+static
+uint32_t
+ds_rt_get_environment_variable (const ep_char16_t *name,
```

Comment:
Do we need a separate command to retrieve a single env var? There is already a command that returns the entire environment block for a process.

Reply:
I suppose we don't NEED it. I just like the symmetry of providing a get/set pair

Reply:
One thing we could do to be more conservative with command ids would be to create one command that takes a bit that determines whether we are getting or setting an env var.

Reply:
If you're worried about IDs I would just get rid of the GetEnvironmentVariable one since we already have the ability to get the whole block like you mention

Reply:
I wouldn't worry about it from the command ID angle but I do suggest we get rid of it if there isn't a clear need for it. All else being equal, I have the general principle less code is better than more code.

Reply:
I just pushed changes to remove the get command